### ISSUE TITLE
why SQLite4Unity3d exists link is broken

### ISSUE THREAD

**Issue by ZackOfAllTrades** [codecoding/SQLite4Unity3d#117]
http://www.codecoding.com/sqlite4unity3d-using-sqlite-net-library-and-unity3d-free-edition/ is a broken link,

Im trying to find what the difference is between this repo and the original.

**Comment by robertohuertasm** [comment]
which one is the original one?

**Comment by AlexSchuetz** [comment]
The website [http://www.codecoding.com/](http://www.codecoding.com/) seems to be down. The last archived content of this link can be found here:

[Archive.org](https://web.archive.org/web/20210127132915/http://www.codecoding.com/sqlite4unity3d-using-sqlite-net-library-and-unity3d-free-edition/)

**Comment by robertohuertasm** [comment]
@AlexSchuetz, yes, the site is down. I decided to discontinue it. I'll move that docs to the new one when I have some time.

**Comment by AlexSchuetz** [comment]
@robertohuertasm Or just include the content of that article in your readme ;)

**Comment by robertohuertasm** [comment]
You're totally right! 🤦🏻‍♂️
